addons/l10n_tr/views/account_move_line_views.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record model="ir.ui.view" id="account_invoice_form">
+            <field name="name">Account Invoice</field>
+            <field name="model">account.move</field>
+            <field name="inherit_id" ref="account.view_move_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='invoice_line_ids']//field[@name='tax_ids']" position="after">
+                    <field name="l10n_tr_available_exception_code_ids" invisible="True"/>
+                    <field name="l10n_tr_exception_code_id"
+                           domain="[('id', 'in', l10n_tr_available_exception_code_ids)]"
+                           attrs="{
+                           'required': [('move_id.country_code', '=', 'TR'),
+                                        ('l10n_tr_available_exception_code_ids','!=',[]),
+                                        ],
+                           'invisible': [('move_id.country_code', '!=', 'TR')]}"
+                    />
```

Comment:
can you inline the `attrs` plz? I know this is more readable like this but this is messing up the view when opening it from the odoo backend